app/src/main/res/layout/activity_home_screen.xml

```diff
@@ -4,17 +4,14 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="horizontal"
     tools:context=".HomeScreenActivity">
 
-
-    <TextView
-        android:id="@+id/app_name"
-        android:text="movIO"
-        android:textSize="35sp"
-        android:textColor="@color/black"
-        android:textStyle="bold"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+    <LinearLayout
```

Comment:
Are you always going to have a movie list inside this container? Maybe the id is not very appropriate.